src/courses/mappers/08.md

```diff
@@ -0,0 +1,285 @@
+---
+order: 8
+next: 09.md
+title: OHDF Schema Review
+author: Charles Hu
+---
+
+## OHDF Schema Review
+
+Now that you've familiarized yourself with the formal OHDF schema, let's revisit the GoSec and Twistlock examples and apply some of that knowledge. 
```

Comment:
Add some more questioning not just reviewing what we've gone over, but also introduce leading questions re multiple fields on either side (need multiple source fields to properly fill an hdf field or a single source field can satisfy multiple hdf fields) and modifying fields (source contains data as string but i need it as int or you only need a substring of the total string)